src/EnergyPlus/HVACCooledBeam.cc

```diff
@@ -210,7 +210,6 @@ namespace HVACCooledBeam {
         static constexpr std::string_view RoutineName("GetCoolBeams "); // include trailing blank space
 
         int CBIndex;                     // loop index
-        int CBNum;                       // current fan coil number
```

Comment:
[src/EnergyPlus/HVACCooledBeam.cc:213]:(style),[variableScope],The scope of the variable 'CBNum' can be reduced.